www/views.py

```diff
@@ -458,3 +458,45 @@ def createuser(request):
     else:
         form = CreateUserForm()
     return render(request, "www/createuser.html", {"userform": form})
+
+@login_required
+@staff_member_required
+def deleteuser(request, id, delete):
+    user = CustomUser.objects.get(id=id)
+    print (user.state)
+    if not (user.state == CustomUser.MEMBER or user.state == CustomUser.RESIGN_DELETE or user.state == CustomUser.RESIGN_SAVE_ACCOUNT):
+        messages.error(
+            request,
+                    _(
+                        "User not member or requesting delete"
+                    ),
+                )
+        return userdetails(request, id)
+
+    custominvoices = CustomInvoice.objects.filter(user=user)
+
+    for invoice in custominvoices:
+        invoice.delete()
```

Comment:
```
CustomInvoice.objects.filter(user=user).delete()
```
Would be faster and in some cases does not require loading all the objects into memory before deletion

Reply:
Done's